projects/storefrontapp/project.json

```diff
@@ -19,6 +19,12 @@
         "polyfills": ["core-js/es/reflect", "zone.js"],
         "tsConfig": "projects/storefrontapp/tsconfig.app.json",
         "assets": [
+          "projects/storefrontapp/src/.well-known/apple-developer-merchantid-domain-association.txt",
+          {
+            "glob": "apple-developer-merchantid-domain-association.txt",
+            "input": "projects/storefrontapp/src/.well-known",
+            "output": "/"
+          },
```

Comment:
[QUESTION]

1. I cannot find those assets locally as they are git-ignored. I guess I'd need to create those files manually myself if I want to use OPF QuickBuy ApplePay?. Shall we document in the code README.md somewhere how I can prepare `.well-known` files for the sake of my local testing?

2. Should customers also prepare such a `.well-known` folder and configure "assets" in their `angular.json` (or `project.json`)?
If yes, then I believe we should at least document it in our official docs. And even better, I guess we should add it via our OPF installation schematics to customer's app. Shouldn't we?

Reply:
[OUTDATED] It will be documented on how to run ApplePay with using our OPF library. 
`apple-developer-merchantid-domain-association.txt` is no longer part of `project.json` by default.